boards/earlgrey-nexysvideo/src/io.rs

```diff
@@ -1,12 +1,17 @@
 use core::fmt::Write;
 use core::panic::PanicInfo;
 use core::str;
+#[cfg(not(test))]
 use kernel::debug;
 use kernel::debug::IoWrite;
+#[cfg(not(test))]
 use kernel::hil::gpio::Configure;
+#[cfg(not(test))]
 use kernel::hil::led;
 
+#[cfg(not(test))]
 use crate::CHIP;
+#[cfg(not(test))]
```

Comment:
maybe: can we move these imports into the the `cfg` guard on line 41? Basically tie them to the `panic_fmt` where they are used rather than a whole bunch of on/offs up here?